aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -83,13 +81,12 @@ Tensor rot90(const Tensor& self, int64_t k, IntList dims) {
     case 1:
       return self.flip({dims[1]}).transpose_(dims[0], dims[1]);
     case 2:
-      return self.flip({dims[0]}).flip({dims[1]});
+      return self.flip(dims);
     case 3:
       return self.transpose(dims[0], dims[1]).flip({dims[1]});
```

Comment:
This can be more efficient if you do `.flip({dims[0]}).transpose_(dims[0], dims[1])`.

Reply:
Excellent! I will use your implementation!